tests/series/is_ordered_categorical_test.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import polars as pl
+import pyarrow as pa
+import pytest
+
+import narwhals as nw
+
+
+def test_is_ordered_categorical() -> None:
+    s = pl.Series(["a", "b"], dtype=pl.Categorical)
+    assert nw.is_ordered_categorical(nw.from_native(s, series_only=True))
+    s = pl.Series(["a", "b"], dtype=pl.Categorical(ordering="lexical"))
+    assert not nw.is_ordered_categorical(nw.from_native(s, series_only=True))
+    s = pl.Series(["a", "b"], dtype=pl.Enum(["a", "b"]))
+    assert nw.is_ordered_categorical(nw.from_native(s, series_only=True))
```

Comment:
we need to decide what to do for Polars Enums here

- `nw.Enum`, and which then just isn't supported for pandas-like and pyarrow
- just map `pl.Enum` to `nw.Categorical`

gonna have to sleep on this one

Reply:
have gone with the second option